main.go

```diff
@@ -63,8 +63,8 @@ func generate(name string, args []string) {
 		api         = flags.String("api", "", "API to generate (e.g., gl)")
 		ver         = flags.String("version", "", "API version to generate (e.g., 4.1)")
 		profile     = flags.String("profile", "", "API profile to generate (e.g., core)")
-		addext      = flags.String("addext", "", "Regular expression of extensions to include (e.g., .*)")
-		remext      = flags.String("remext", "", "Regular expression of extensions to exclude (e.g., .*)")
+		addext      = flags.String("addext", "", "If non-empty, a regular expression describing which extensions to include in addition to those supported by the selected profile")
```

Comment:
It seems important to include, since it affects behavior.

I think output of `-help` should prioritize clarity over brevity. Look at `go generate -help`, it's more than 100 lines of output.